nvtabular/ops/operator.py

```diff
@@ -26,6 +27,19 @@
 ColumnNames = List[Union[str, List[str]]]
 
 
+class Supports(Flag):
+    """ Indicates what type of data representation this operator supports. """
+
+    # cudf dataframe
+    CPU_DATAFRAME = auto()
+    # pandas dataframe
+    GPU_DATAFRAME = auto()
+    # dict of column name to numpy array
+    CPU_ARRAY = auto()
```

Comment:
The names of these array constants seem a little confusing, because the format is actually a `dict`. Maybe `CPU_DICT_ARRAY` or something like that?